.travis.yml

```diff
@@ -9,6 +9,11 @@ script:
 jdk:
   - oraclejdk8
 
+stages:
+  - name: test
+  - name: release
+    if: (branch = master) AND (type = push)
```

Comment:
This seems to be inconsistent with `if [[ "$TRAVIS_TAG" ]];` below?

Reply:
Good catch! This is copied from the metals repo. I removed the `branch = master` guard but kept `type = push`